docs/higal/higal.rst

```diff
@@ -0,0 +1,51 @@
+.. doctest-skip-all
+
+.. _astroquery.higal:
+
+**************************************
+HiGal Image Cutout and Catalog service
+**************************************
+
+Getting started
+===============
+
+The HiGal image cutout service is a sub-service of the Space Science Data
+Center Sky Explorer (https://tools.ssdc.asi.it/).  It enables image cutout
+queries and catalog queries.
+
+An example catalog query:
+
+.. python::
```

Comment:
either remove this (should work without) or add `.. code-block:: python`

Reply:
Changed to `.. code-block:: python`

Reply:
Added blank line after this line